owncloudData/src/main/java/com/owncloud/android/data/spaces/datasources/implementation/OCLocalSpacesDataSource.kt

```diff
@@ -137,6 +141,44 @@ class OCLocalSpacesDataSource(
                 webDavUrl = webDavUrl
             )
 
+        @VisibleForTesting
+        fun SpacesEntity.toModel() =
+            OCSpace(
```

Comment:
The only difference between this one and `SpacesWithSpecials.toModel()` is that we have specials on one of them. But the "main" parsing could be shared so if we need to apply any change to one class, we won't need to update 2 parsers. 

Is there a way to do this?

Reply:
I realised, and I thought I could reuse the `SpacesEntity.toModel()` in the other method, but in `OCSpace`, the `special` property is a val, and cannot be reassigned. I don't know if it is worth it to make it a var so that we can reuse the previous method when we are not going to reassign it anywhere else for the moment... My decision was not to do it 😬, what do you think?

Reply:
I've just pushed a commit updating this https://github.com/owncloud/android/pull/3875/commits/0f717e64ca9479ef2995907e329f3ca71786510c Check it out 👍